test/regression-c-semantics/test-regression-c-semantics.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+$KORE_PARSER --no-print-definition --no-print-pattern definition.kore --pattern executable.kore --module C
```

Comment:
I thought we need to specify the module with = (`--module=C`)

Reply:
Both are accepted.